docs/notebooks/demo/gptx-healthcare/gretel-demo-gptx-healthcare.ipynb

```diff
@@ -75,63 +92,7 @@
    "source": [
     "## Loading dataset\n",
     "\n",
-    "For the proof-of-concept, we work with summarized notes, since samples in the original dataset can be quite long. We call Haiku to generate a summarized version of the first 10,000 samples in the dataset.\n",
-    "\n",
-    "If you prefer, you can skip the next cell and run the one just after, which will load the summarized dataset from a S3 bucket."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "## SKIP TO DOWNLOAD RESULTS DIRECTLY\n",
-    "\n",
-    "from glob import glob\n",
-    "from tqdm import tqdm\n",
-    "\n",
-    "import pandas as pd\n",
-    "from datasets import load_dataset\n",
-    "\n",
-    "\n",
-    "# Load original dataset\n",
-    "original_dataset = load_dataset(\"AGBonnet/augmented-clinical-notes\")\n",
-    "\n",
-    "# Generate summaries\n",
-    "for i, s in tqdm(enumerate(original_dataset[\"train\"][:10_000][\"conversation\"])):\n",
-    "    inst = f\"Generate a shorter version of the following conversation.\"\n",
-    "    summary = call_bedrock(message=s, system=inst, model_id=BASE_MODEL_ID)\n",
-    "    with open(f'summaries/transcript_{i:05d}.txt', 'w') as fid:\n",
-    "        fid.write(summary)\n",
-    "\n",
-    "# Load and process notes and transcripts\n",
-    "notes_files = glob('summaries/note*')\n",
-    "transcripts_files = glob('summaries/transcript*')\n",
-    "\n",
-    "notes = []\n",
-    "for f in sorted(notes_files):\n",
-    "    if '.txt' in f:\n",
-    "        with open(f) as fid:\n",
-    "            content = fid.read()\n",
-    "            notes.append(content.split('\\n')[-1])\n",
-    "\n",
-    "transcripts = []\n",
-    "for f in sorted(transcripts_files):\n",
-    "    if '.txt' in f:\n",
-    "        with open(f) as fid:\n",
-    "            content = fid.read()\n",
-    "            transcript = [r for r in content.split('\\n') if r.strip()]\n",
-    "            if 'short' in transcript[0]:  # remove intro if any\n",
-    "                transcript = transcript[1:]\n",
-    "            transcripts.append('\\n'.join(transcript))\n",
-    "\n",
-    "# Add notes and transcripts to dataframe\n",
-    "summaries = pd.DataFrame(columns=['conversation', 'note'])\n",
-    "summaries['conversation'] = transcripts\n",
-    "summaries['note'] = notes\n",
-    "\n",
-    "summaries.to_csv('summaries.csv')"
+    "For the proof-of-concept, we work with summarized notes, since samples in the original dataset can be quite long. We call Haiku to generate a summarized version of the first 10,000 samples in the dataset. You can download the summarized dataset from a S3 bucket, below."
    ]
   },
   {
```

Comment:
Did we want to upload this summaries.csv.gz to HF? Seems broadly helpful to the community